api/src/resources/comment.rs

```diff
@@ -498,7 +512,7 @@ pub struct EntityKind(pub String);
 impl FromStr for EntityKind {
     type Err = Error;
 
-    fn from_str(string: &str) -> Result<Self, Error> {
+    fn from_str(string: &str) -> Result<Self> {
```

Comment:
Why is this no longer with an `Error`?

Reply:
At the top of the file, bringing
```rust
crate::errors::{Error, Result};
```
into scope shadows the default meaning of `Result` from the standard library. So it is now defined as
```rust
pub type Result<T> = StdResult<T, Error>;
```